Sources/Sentry/SentryIOTrackingIntegration.m

```diff
@@ -0,0 +1,27 @@
+#import "SentryIOTrackingIntegration.h"
+#import "SentryNSDataSwizzling.h"
+#import "SentryOptions.h"
+
+@interface
+SentryIOTrackingIntegration ()
+
+@property (nonatomic, strong) SentryOptions *options;
+@end
+
+@implementation SentryIOTrackingIntegration
+
+- (void)installWithOptions:(SentryOptions *)options
+{
+    self.options = options;
+    if (options.enableSwizzling && options.enableAutoPerformanceTracking
+        && options.enableIOTracking) {
```

Comment:
`l`: Can we maybe check each flag individually as we do it in SentryNetworkTrackingIntegration
https://github.com/getsentry/sentry-cocoa/blob/229c663f6bdf225f8ef855f8355d48196a022cb8/Sources/Sentry/SentryNetworkTrackingIntegration.m#L13-L18

The code coverage will immediately tell us then if we forgot to write a test for a flag. As we definitely don't want to enable this integration by accident I th9nk it's a good idea.

Reply:
Sure, good point.